sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -671,3 +671,7 @@ ALTER TABLE `insurance_companies` ADD `cqm_sop` int DEFAULT NULL COMMENT 'HL7 So
 #IfNotRow2D list_options list_id order_type option_id order
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('order_type','order','Order',90,0);
 #EndIf
+
+#IfNotColumnType procedure_type procedure_code varchar(63)
+ALTER TABLE `procedure_type` MODIFY `procedure_code` varchar(63) NOT NULL DEFAULT '' COMMENT 'code identifying this procedure',
```

Comment:
varchar(64) in keeping with size I changed procedure_order_code (L-530) procedure_code. 
Nitpicking I guess and won't matter in the end